### ISSUE TITLE
Mac: Text replacement not working.

### ISSUE THREAD

**Issue by sushib** [microsoft/vscode#224082]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: Version: 1.91.1 (Universal)
- OS Version: 14.5 (23F79)

Steps to Reproduce:

Steps to Reproduce:
Go to the Keyboard section of the MacOS settings menu
Navigate to the 'Text' section
Hit the '+' button and add a key/phrase that the system will then replace with whichever character/phrase you like
Use the phrase to be expanded in VS Code
Nothing happens.

**Comment by amirho1** [comment]
Hi, body @sushib  


it seems that it only works on the keynote 


<img width="765" alt="Screenshot 2024-07-28 at 10 07 34 AM" src="https://github.com/user-attachments/assets/566000cd-45bd-48cd-b242-fb8840f68075">

**Comment by sushib** [comment]
HiIt works on all the Mac applications. Do you have a Mac ?Cheers BrunoOn 28 Jul 2024, at 8:43 AM, Arash ***@***.***> wrote:﻿
Hi, body @sushib
it seems that it only works on the keynote
Screenshot.2024-07-28.at.10.07.34.AM.png (view on web)

**Comment by amirho1** [comment]
@sushib Yes I do have a Mac, and as you said it is working on most places

**Comment by sushib** [comment]
I am speaking about this feature of Mac not working into visual code



> On 28 Jul 2024, at 09:57, Arash ***@***.***> wrote:
> 
￼￼

**Comment by karladler** [comment]
can confirm the problem

**Comment by SaintPepsi** [comment]
Also having this problem. Mac's Text replacement works everywhere seemling anywhere there's a "regular input" for example it doesn't work in VSCode or Google chrome URL when you're typing.
